src/partitioned-dbs/index.rst

```diff
@@ -0,0 +1,384 @@
+are free to use the normal ``/dbname/_all_docs`` to read documents from
+multiple partitions.
+
+Next, we'll create a design document containing our index for
+getting all readings from a given sensor. The map function is similar to
+our earlier example except we've accounted for the change in the document
+id.
+
+.. code-block:: javascript
+
+    function(doc) {
+        if(doc._id.indexOf(":sensor-reading-") < 0) {
+            return;
+        }
+        for(var r in doc.readings) {
+            emit([doc.sensor_id, r[0]], r[1])
+        }
+    }
+
+We can upload our design document and try out a partitioned
+query:
+
+.. code-block:: bash
+
+    shell> cat ddoc.json
+    {
+        "_id": "_design/sensor-readings",
+        "views": {
+            "by_sensor": {
+                "map": "function(doc) { ... }"
+            }
+        }
+    }
+    shell> curl http://127.0.0.1:5984/my_new_db/_partition/sensor-260/_design/sensor-readings/_view/by_sensor
+    {"total_rows":4,"offset":0,"rows":[
+    {"id":"sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf","key":["sensor-260","0"],"value":null},
+    {"id":"sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf","key":["sensor-260","1"],"value":null},
+    {"id":"sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf","key":["sensor-260","2"],"value":null},
+    {"id":"sensor-260:sensor-reading-ca33c748-2d2c-4ed1-8abf-1bca4d9d03cf","key":["sensor-260","3"],"value":null}
+    ]}
+
+Hooray! Our first partitioned query. For experienced users, that may not
+be the most exciting development, given that the only things that have
+changed are a slight tweak to the document id, and accessing views with
+a slightly different path. However, for anyone who likes performance
+improvements, its actually a big deal. By knowing that the view results
+are all located within the provided partition name, our partitioned
+queries now perform nearly as fast as document lookups!
+
+The last thing we'll look at is how to query data across multiple partitions.
+For that, we'll implement the example sensors by field query from our
+initial example. The map function will use the same update to account
+for the new document id format, but is otherwise identical to the previous
+version:
+
+.. code-block:: javascript
+
+    function(doc) {
+        if(doc._id.indexOf(":sensor-reading-") < 0) {
+            return;
+        }
+        emit(doc.field_name, doc.sensor_id)
+    }
+
+Next we'll create a new design doc with this function. Be sure to notice
+that the ``"options"`` member contains ``"partitioned": false``.
+
+.. code-block:: bash
+
+    shell> cat ddoc2.json
+    {
+      "_id": "_design/all_sensors",
+      "options": {
+        "partitioned": false
+      },
+      "views": {
+        "by_field": {
+          "map": "function(doc) { ... }"
+        }
+      }
+    }
+    shell> $ curl -X POST -H "Content-Type: application/json" http://127.0.0.1:5984/my_new_db -d @ddoc2.json
```

Comment:
Here you show the ddoc upload. Go figure. :stuck_out_tongue:

Reply:
Addressed in 347a6ce.